sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonHighlighter.java

```diff
@@ -120,4 +136,9 @@ private void highlight(Token token, TypeOfText typeOfText) {
     newHighlighting.highlight(tokenLocation.startLine(), tokenLocation.startLineOffset(), tokenLocation.endLine(), tokenLocation.endLineOffset(), typeOfText);
   }
 
+  private static boolean isDocString(AstNode stringNode) {
+    AstNode statement = stringNode.getFirstAncestor(PythonGrammar.STATEMENT);
+    return statement != null && statement.equals(statement.getParent().getFirstChild(PythonGrammar.STATEMENT));
```

Comment:
@vilchik-elena OK. I made an effort to squeeze (info rebase) the commits in order to shorten up the history...